SCF-Session-5/lcmhcf/LcmHcf.java

```diff
@@ -0,0 +1,44 @@
+package lcmhcf;
+
+/**
+ * This class contains methods to find LCM and HCF of two numbers using recursion.
+ * @author Ishita Mathur
+ *
+ */
+public class LcmHcf 
```

Comment:
Class and method names `LcmHcf`, `lcm` & `hcf`.

Reply:
class name can't be changed now but I have updated the method names and will take care about this in future.